hw3/01-characters.js

```diff
@@ -1,4 +1,88 @@
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
+const charactersContainer = document.querySelector('.characters-container');
 
-fetch(url);
+const createRow = function createGenericImgRowForDOM(rowIndex) {
+  const newRow = document.createElement('div');
+
+  newRow.className = 'row row-cols-2 row-cols-lg-4 align-self-center';
+  newRow.id = `row-${rowIndex}`;
+
+  charactersContainer.append(newRow);
+};
+
+const createImg = function createGenericImgForDOM(rowIndex, imgIndex) {
+  const rowContainer = document.getElementById(`row-${rowIndex}`);
+  const newCol = document.createElement('div');
```

Comment:
Can you elaborate on the rowContainer and newCol variables? I'm may be misinterpreting their purpose.

Reply:
After inspecting the code in the browser, it looks like you're using Bootstrap grid but on groups of pictures. Although this works just as intended, I would recommend simply using flexbox where you have the parent 'container' and each photo would be an item in that container. Then in the CSS file, you can target the parent container and display flex. Therefore, you can get the same results but with less JS code!

Reply:
I think my reasoning behind that was that I wanted a clear cutoff between screen sizes, so that rows would be limited to 2 images on mobile/smaller screens, and 4 otherwise.